source/04-1_alignak_configuration/core_configuration.rst

```diff
@@ -1,64 +1,41 @@
-.. _configuration/arbiter:
+.. _configuration/core:
 
-=======
-Arbiter
-=======
+==================
+Core configuration
+==================
+
+The core configuration part describes the Alignak framework infrastructure (which daemons are used and how they are). 
+
+For historical reasons, the configuration is splited in two kinds of files :
+
+    * *.cfg* files for storing settings more binded to the role of the daemon concerned.
+    * *.ini* files for storing settings considered as parameters
 
+In a feature paradise, one .cfg file per daemon should be enougth to configure all settings of the concerned daemon.
```

Comment:
we may precise: "Currently each daemon has a .cfg and a .ini file for its configuration. The ini file contains the parameters use by the daemon to communicate with the other daemons in the Alignak framework where as the .cfg file contains parameters dedicated to its role  in the whole stuff"